### PR TITLE
stake-pool: seed suffix to allow creating & discarding multiple transient stake accs

### PR DESCRIPTION
**Pull request by jon-chuang**
We are just adding struct fields to ensure backwards compatibility for future work to get this working.

### CODE REVIEW

File: stake-pool/program/proptest-regressions/state.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 cc 41ce1c46341336993e5e5d5aa94c30865e63f9e00d31d397aef88ec79fc312ca
+cc 2e5aed24877ce7b166d018e73d8617f51ec250b8ef51a9f539a978ef99b3304b # shrinks to test_amount = 1
```
Comment (joncinque) [inline stake-pool/program/proptest-regressions/state.txt:2]: is this needed?

**Review by joncinque — COMMENTED** [review]
just a nit, otherwise this is good to go!

File: stake-pool/program/proptest-regressions/state.txt (hunk 1 above)
Comment (jon-chuang) [reply stake-pool/program/proptest-regressions/state.txt]: not sure what is or isn't needed when it comes to proptests actually...

**Comment by joncinque** [comment]
Looks great, thanks!

**Pull request merged**